test/hooks/test-trace-restify.js

```diff
@@ -26,7 +26,7 @@ var versions = {
 
 var server;
 
-Object.keys(versions).forEach(function(version) {
+Object.keys(versions).forEach(function(version, i) {
```

Comment:
Do you use the index anywhere? It is hard to search for `i` – I can't find it by eyeballing the code.